tests/xtro-sharpie/common-CoreMotion.ignore

```diff
@@ -2,3 +2,10 @@
 
 ## no generator support for FastEnumeration - https://bugzilla.xamarin.com/show_bug.cgi?id=4391
 !missing-protocol-conformance! CMSensorDataList should conform to NSFastEnumeration
+
+## For some reason sharpie is not picking up the following classes
```

Comment:
double check the umbrella header, it's likely just missing the new header file
if that's the case file a rdar - if not let's figure out why and file the rdar next ;-)

Reply:
Oh wow @spouliot, nice catch, `CMMovementDisorderManager.h` is indeed missing from the umbrella header. TIL ❤️